File: Maintix_API/Models/Usuario.cs
namespace Maintix_API.Models
{
    public class Usuario
    {
        public int Id { get; set; }
        public int TipoUsuarioId { get; set; }
        public string Email { get; set; } = string.Empty;
        public string Passwd { get; set; } = string.Empty;
    }
}

File: mobile/app/src/main/java/com/tonhete/maintixapp/data/models/Usuario.kt
package com.tonhete.maintixapp.data.models
import com.google.gson.annotations.SerializedName

data class Usuario(
    @SerializedName("id") val id: Int,
    @SerializedName("tipoUsuarioId") val tipoUsuarioId: Int,
    @SerializedName("email") val email: String,
    @SerializedName("passwd") val passwd: String?
)